docs/source/intro/tutorial.md

```diff
@@ -501,22 +501,42 @@ class MultiplyAddWorkChain(WorkChain):
 ```
 
 :::{note}
-Besides WorkChain's, workflows can also be implemented as *work functions*.
+Besides WorkChain's, workflows can also be implemented as [*work functions*](https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/workflows/concepts.html).
 These are ideal for workflows that are not very computationally intensive and can be easily implemented in a Python function.
 :::
 
 Let's run the `WorkChain` above!
+
+:::{note}
+To simplify the basic tutorial, we have left out the steps to register the `MultiplyAddWorkChain` as a plugin. This allows us to access it through the [*entry point*](https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/plugins.html#topics-plugins-entrypoints) named `core.arithmetic.multiply_add`.
+
+```console
+$ verdi plugin list
+```
+to show the available pulgins and their entry points.
+:::
+
 Start up the `verdi shell` and load the `MultiplyAddWorkChain` using the `WorkflowFactory`:
 
 ```{code-cell} ipython3
 from aiida import plugins
 MultiplyAddWorkChain = plugins.WorkflowFactory('core.arithmetic.multiply_add')
 ```
 
-The `WorkflowFactory` is a useful and robust tool for loading workflows based on their *entry point*, e.g. `'core.arithmetic.multiply_add'` in this case.
+The `WorkflowFactory` is a useful and robust tool for loading workflows based on their [*entry point*](https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/plugins.html#topics-plugins-entrypoints), e.g. `'core.arithmetic.multiply_add'` in this case.
+
+:::{note}
+In AiiDA, an [entry point](https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/plugins.html#topics-plugins-entrypoints) is a way to specify and register a plugin or code so that it can be discovered and used by the AiiDA framework.
```

Comment:
```suggestion
The [entry point](https://aiida.readthedocs.io/projects/aiida-core/en/latest/topics/plugins.html#topics-plugins-entrypoints) is a way to make it easily access a core or user-defined AiiDA object and make the AiiDA objects discoverable. 
```

Reply:
I'd also prefer this is not as a "Note", it just has too many "Note" in this section, and it is not something the reader need to note but more like detailed declarative information.